src/CameraControls.ts

```diff
@@ -3214,6 +3220,19 @@ export class CameraControls extends EventDispatcher {
 
 	protected _removeAllEventListeners(): void {}
 
+	// Applies and resets focal offsets to play nice with lookAt and zoom-to-cursor
+	// https://github.com/yomotsu/camera-controls/issues/491
+	_resetFocalOffsets() {
```

Comment:
let's make this as a private method at this moment
```suggestion
	protected _resetFocalOffsets() {
```